src/components/SettingsLoader.vue

```diff
@@ -1,18 +1,52 @@
 <template>
     <div id="settings-loader">
-        <slot v-if="isLoaded" />
+        <slot v-if="phase === PHASES.LOADED" />
 
-        <div v-else :class="['modal', 'z-top', {'is-active': error}]">
+        <div v-if="phase > PHASES.LOADED" class="modal z-top is-active">
             <div class="modal-content">
                 <div class="notification is-danger">
                     <h3 class="title">Error</h3>
                     <h5 class="title is-5">{{error && error.name}}</h5>
                     <p>{{error && error.message}}</p>
-                    <div v-if="error && error.solution">
-                        <br/>
-                        <h5 class="title is-5">Suggestion</h5>
-                        <p>{{error && error.solution}}</p>
+                    <br />
+                    <h5 class="title is-5">Suggestion</h5>
+
+                    <p v-if="phase === PHASES.GAME_FAILED">
+                        This is a problem with the mod manager itself.
+                        If there's a newer version of the manager
+                        available, try installing it.
+                    </p>
+
+                    <div v-else-if="phase === PHASES.SETTINGS_FAILED">
+                        <p>
+                            Loading of local user settings failed. You
+                            can use the button below to reset the
+                            settings, but note that all settings for all
+                            games will be lost and this can't be undone.
+                            This won't affect the loaded mods or mod profiles.
+                        </p>
+                        <br />
+                        <button @click="resetSettings" class="button is-white">
+                            Reset settings
+                        </button>
                     </div>
+
+                    <p v-else-if="phase === PHASES.RESET_FAILED">
+                        Well, that's awkward. Resetting of the settings
+                        failed as well. You can still try to reset the
+                        settings manually by following these
+                        <a target="_blank" rel="noopener noreferrer"
```

Comment:
Any reason for noopener / noreferrer?

Can we bind the Link component provider earlier (if it needs binding?) as that can open in an actual browser

Reply:
> Any reason for noopener / noreferrer?

Generally good practice, there are some security concerns with links that don't use these.

Not sure about the Link component suggestion, leaving that up to @anttimaki

Reply:
My idea was that the SettingsLoader mounts before any providers etc. to reduce points of failure, and to ensure that any problem encountered is actually related to loading the settings. If having the link open in a browser is considered worth it to introduce the LinkingProvider earlier, and the LinkingProvider is seen as a stable and tested implementation, I see no reason why it couldn't be done. Personally I don't see it fitting into the idea (that I had) of the SettingsLoader, though.

Reply:
That makes sense to me at least, given that the goal was to minimize the points of failure for this component.

Reply:
I think it’d be good to do, and it’s stable as in it only depends on external libraries for opening, it’s not reliant on settings or anything user accessible

Reply:
This is now [passed to SettingsLoader as a prop](https://github.com/ebkr/r2modmanPlus/pull/917/commits/8cacfe28b284832caa85493bd2a01f12b30d59cf#diff-e7478582172dd38f0de771e700456eb24487444bd9553f2eff6e2b4ec0b4fd86) to keep things consistent with `logError`. Also skipped the provider and used the implementation directly so that all the providers are initiated in the same place (`App.vue`).